### PR TITLE
fix typo in help text

### PR DESCRIPTION
**Pull request by dooglus**
'in which the transaction is included in'

### CODE REVIEW

File: src/rpc/rawtransaction.cpp [hunk 1]
```diff
@@ -211,7 +211,7 @@ UniValue gettxoutproof(const UniValue& params, bool fHelp)
             "\nNOTE: By default this function only works sometimes. This is when there is an\n"
             "unspent output in the utxo for this transaction. To make it always work,\n"
             "you need to maintain a transaction index, using the -txindex command line option or\n"
-            "specify the block in which the transaction is included in manually (by blockhash).\n"
+            "specify the block in which the transaction is included manually (by blockhash).\n"
```
Comment (btcdrak) [inline src/rpc/rawtransaction.cpp]: maybe it needs a comma `block in which the transaction is included in, manually`. The whole sentence could do with being rewritten, it's very laboured language.

Comment (dooglus) [inline src/rpc/rawtransaction.cpp]: Maybe. My point is that the word 'in' is there twice and shouldn't be.

Either

> "the block which the tx is included **in**"

or

> "the block **in** which the tx is included"

are OK but

> "the block **in** which the tx is included **in**"

isn't.

Comment (btcdrak) [inline src/rpc/rawtransaction.cpp]: Agreed. Overall the entire NOTE is painful for me to read.

Saying thing like "By default this function only works sometimes" and "To make it always work" is pretty strange. There must be a better way to express this note than flail about like that.

Comment (dooglus) [inline src/rpc/rawtransaction.cpp]: > NOTE: By default this function only works sometimes. This is when there is an unspent output in the utxo for this transaction. To make it always work, you need to maintain a transaction index, using the -txindex command line option or specify the block in which the transaction is included in manually (by blockhash).

I think this is what the note is trying to say:

This function only works when the transaction(s) can be located. Transactions can be located if any of these conditions are met:

1) the transaction has at least one unspent output
2) the transaction index is active (enabled using the -txindex command line option)
3) the optional blockhash parameter is provided and the corresponding block contains the transaction

Aside: why does the function accept multiple txids but only one blockhash? It appears that for this function to work all the transactions have to be in the same block. In which case it should be possible to find their containing block if any one of them has an unspent output. But the code appears to only attempt to look up the last specified txid. But this is way outside the scope of this typo-fixing issue!

**Comment by laanwj** [comment]
Yes, let's improve the whole text, adding one word here is lipstick on a pig.

**Pull request merged**